circle.yml

```diff
@@ -645,26 +669,19 @@ jobs:
           command: RUSTFLAGS="-Z sanitizer=address -C opt-level=0" cargo +nightly test --target x86_64-unknown-linux-gnu
 
   bindings-rust-coverage:
-    docker:
-      - image: rust:1-buster
+    executor: rust
     steps:
+      - rust_restore_cargo_cache
+      - rust_install_system_dependencies
+      - rust_install_nightly
       - checkout
-      - run:
-          name: Update environment
-          command: |
-            echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
-            apt -qq update
-            apt -yq -t buster-backports install cmake --no-install-recommends
-            apt -yq install llvm-8-dev clang-8 --no-install-recommends
-            rustup toolchain install nightly-x86_64-unknown-linux-gnu
-            rustup component add llvm-tools-preview
-            cargo install cargo-binutils rustfilt
       - run:
           name: Build (debug mode)
           command: RUSTFLAGS="-Z instrument-coverage" cargo +nightly build --target x86_64-unknown-linux-gnu
```

Comment:
I think `--target x86_64-unknown-linux-gnu` can be removed, but it does not hurt. It was needed in chisel because it was also built to the wasm target.